vars/main.yml

```diff
@@ -28,11 +28,14 @@ synchronous_node_count: 1  # number of synchronous standby databases
 
 # Load Balancing
 with_haproxy_load_balancing: false  # or 'true' if you want to install and configure the load-balancing
+with_haproxy_direct_connections: false # or 'true' if you want to enable haproxy ports that skip pgbouncer (and if pgbouncer_install is 'true')
 haproxy_listen_port:
   master: 5000
+  master_direct: 6000
   replicas: 5001
   replicas_sync: 5002
   replicas_async: 5003
+  replicas_direct: 6001
   stats: 7000
```

Comment:
Let's simplify it a bit, by default we comment out the '_direct' variables 

for example
```suggestion
# Load Balancing
with_haproxy_load_balancing: false  # or 'true' if you want to install and configure the load-balancing
haproxy_listen_port:
  master: 5000
  replicas: 5001
  replicas_sync: 5002
  replicas_async: 5003
# The following ('_direct') ports are used for direct connections to the PostgreSQL database, bypassing the PgBouncer connection pool.
# Uncomment the relevant lines if you need to set up direct connections.
#  master_direct: 6000
#  replicas_direct: 6001
#  replicas_sync_direct: 6002
#  replicas_async_direct: 6003
  stats: 7000
```

and add the appropriate section in the configuration only if the variable is declared, example:

```jinja

{% if pgbouncer_install | bool and haproxy_listen_port.master_direct | default('') | | length > 0 %}
```

And so it is for each variable.

Also note that I suggest adding sync and async direct ports. By default, they are commented out so this is definitely not necessary for everyone, but for those who need it, they can uncomment, for example, only 'master_direct' if they need direct connections only to the master.

Reply:
Thanks, that's perfectly sensible, I've pushed the change. 

Not sure if we should define separate `haproxy_maxconn` values for new cases as well? Although current default values seem to be set high enough so I guess there's no need for that.

Reply:
it is always individual, for some it is a lot and for others it is not enough. In any case, the variable for regulation is available and those who need it will be able to raise it.

Reply:
>  I've pushed the change.

Have you tested this?

Reply:
I [tested](https://github.com/vitabaks/postgresql_cluster/pull/549#issuecomment-1888858062) and made a small [fix](https://github.com/vitabaks/postgresql_cluster/pull/549/commits/1604ea751f86dc2297676ed38242953b97b1b900). 

If you don't have any more add-ons, we can merge this MR, let us know if you're done.